tls/s2n_client_hello.c

```diff
@@ -287,6 +287,40 @@ int s2n_handshake_status_handler(struct s2n_connection *conn)
 
     return 0;
 }
+
+/* in TLS 1.3, pick a signature algorithm scheme based on server preference order along with available cert types */
+int s2n_choose_tls13_sig_scheme_and_set_cert(struct s2n_connection *conn, struct s2n_sig_scheme_list *peer_pref_list,
+                                struct s2n_signature_scheme *sig_scheme_out)
+{
+    /* use server preference list */
+    for (int i = 0; i < s2n_tls13_sig_scheme_pref_list_len; i++) {
```

Comment:
non-blocker: not a huge fan of having this as a global variable, but will assume it will be a struct field in the configured "signature scheme preferences" struct in the future.